atomic_reactor/plugins/pre_add_yum_repo_by_url.py

```diff
@@ -73,24 +74,24 @@ def run(self):
             self.log.info("Skipping add yum repo by url: unsupported for FROM-scratch images")
             return
 
-        if self.repourls:
+        if self.repourls and not is_scratch_build():
             self.validate_yum_repo_files_url()
 
-            for repourl in self.repourls:
```

Comment:
Can we guarantee that `self.repourls` always contain something iterable?

Reply:
There are validations in place to check if that is a list in case it is defined. Do you have any case in mind that would break due to the de-indentation?

Reply:
No, I'm just asking to be sure